pkg/engine/update.go

```diff
@@ -186,7 +218,7 @@ func installPlugins(
 	return allPlugins, defaultProviderVersions, nil
 }
 
-func installAndLoadPolicyPlugins(plugctx *plugin.Context, policies []RequiredPolicy, localPolicyPackPaths []string,
+func installAndLoadPolicyPlugins(plugctx *plugin.Context, policies []RequiredPolicy, localPolicyPackPaths []LocalPolicyPack,
```

Comment:
We should rename `localPolicyPackPaths` to be `localPolicyPacks`, as it is now not just a string slice of file paths, but our proper structs that include more information. (And given the places where we are converting one to the other, being clear is kinda important.)